packages/artillery-plugin-ensure/index.js

```diff
@@ -65,29 +66,45 @@ class EnsurePlugin {
         global.artillery.globalEvents.emit('checks', checkTests);
 
         checkTests
-          .sort((a, b) => (a.result < b.result) ? 1 : -1)
+          .sort((a, b) => (a.result < b.result ? 1 : -1))
           .forEach((check) => {
-          if (check.result !== 1) {
-            global.artillery.log(
-              `${chalk.red('fail')}: ${check.original}${check.strict ? '' : ' (optional)'}`
-            );
-            if (check.strict) {
-              global.artillery.suggestedExitCode = 1;
+            if (check.result !== 1) {
```

Comment:
ignore this change, it's just prettier. Prettier was failing on this file before due to a linting config problem, so it hasn't been applied here for a while